x/ibc/03-connection/keeper/keeper_test.go

```diff
@@ -355,9 +354,9 @@ func (chain *TestChain) updateClient(client *TestChain) {
 
 func (chain *TestChain) createConnection(
```

Comment:
nit: idk why we're writing function signatures like this? Just make the args a single line.